SeleniumTestXUnit/Tests/Features/Item/AddBelow.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Add Note Below
+    As a logged in user, I want to be able to create a note below another
+
+
+    Background:
+        Given the user is logged in
+        And the user has existing notes
+
+    @acceptance
+    Scenario: Add below
+        Given the user wants to add a new note below an existing one
+        When the user hovers over the desired note
```

Comment:
As I noted to your teammates, lines 12-14 should be only one step that hovers over a note, expands the menu and selects an option